src/index.html

```diff
@@ -1,14 +1,64 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#">
+          <img src="./images/logo.png" alt="MOYO logo of technic store">
+        </a>
+        <nav class="nav">
+          <ul class="nav-order">
+
+            <li class="nav-element">
+              <a href="#" class="is-active nav-link">Apple</a>
+            </li>
+
+            <li class="nav-element">
+              <a href="#" class="nav-link">Samsung</a>
+            </li>
+
+            <li class="nav-element">
+              <a href="#" class="nav-link">Smartphones</a>
+            </li>
+
+            <li class="nav-element">
+              <a href="#" class="nav-link"
+              data-qa="hover">Laptosps & Computers</a>
+            </li>
+
+            <li class="nav-element">
+              <a href="#" class="nav-link">Gadgets</a>
+            </li>
+
+            <li class="nav-element">
+              <a href="#" class="nav-link">Tablets</a>
+            </li>
+            <li class="nav-element">
+              <a href="#" class="nav-link">Photo</a>
+            </li>
+            <li class="nav-element">
+              <a href="#" class="nav-link">Video</a>
+            </li>
+
```

Comment:
Don't forget to fix comments in all cases
```suggestion

            <li class="nav-element">
              <a href="#" class="nav-link">Tablets</a>
            </li>
            
            <li class="nav-element">
              <a href="#" class="nav-link">Photo</a>
            </li>
            
            <li class="nav-element">
              <a href="#" class="nav-link">Video</a>
            </li>
```